modules/cluster-logging-collector-log-forwarding-about.adoc

```diff
@@ -0,0 +1,122 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-external.adoc
+
+[id="cluster-logging-collector-log-forwarding-about_{context}"]
+= About forwarding logs to third-party systems
+
+Forwarding cluster logs to external third-party systems requires a combination of _outputs_ and _pipelines_ specified in a `ClusterLogForwarder` custom resource (CR) to send logs to specific endpoints inside and outside of your {product-title} cluster. 
+
+An _output_ is the destination for log data and a _pipeline_ defines simple routing from one source to one or more outputs. 
+
+An output can be:
+
+* `default` to use the internal {product-title} Elasticsearch instance. You do not need to configure the default output. If you do configure a `default` output, you will receive an error message because the `default` output is reserved for the Cluster Logging Operator.
```

Comment:
'default' is a built-in output _name_, used in the outputRefs section of a pipeline. The others are output _types_, used in user-defined outputs, which have user-chosen names. Need to separate them somehow.